src/prefect/runtime/task_run.py

```diff
@@ -98,35 +108,53 @@ def get_run_count() -> int:
         return task_run_ctx.task_run.run_count
 
 
-def get_name() -> Optional[str]:
+def get_name() -> str | None:
     task_run_ctx = TaskRunContext.get()
     if task_run_ctx is None:
         return None
     else:
         return task_run_ctx.task_run.name
 
 
-def get_task_name() -> Optional[str]:
+def get_task_name() -> str | None:
     task_run_ctx = TaskRunContext.get()
     if task_run_ctx is None:
         return None
     else:
         return task_run_ctx.task.name
 
 
-def get_parameters() -> Dict[str, Any]:
+def get_parameters() -> dict[str, Any]:
     task_run_ctx = TaskRunContext.get()
     if task_run_ctx is not None:
         return task_run_ctx.parameters
     else:
         return {}
 
 
-FIELDS: dict[str, Callable[[], Any]] = {
+def get_task_run_api_url() -> str | None:
+    if (api_url := get_current_settings().api.url) is None:
+        return None
+    if (task_run_id := get_id()) is None:
+        return None
+    return f"{api_url}/runs/task-run/{task_run_id}"
```

Comment:
i'm not opposed to this but i am curious - what's the use of this one?

Reply:
like why expose `api_url`? it was purely for consistency, although yeah i cannot think of a good reason I'd need it

Reply:
no worries, was just curious!